doc/user-guide/cli.md

```diff
@@ -207,13 +211,12 @@ ploomber-cloud deploy
 The command-line interface will automatically read and encrypt your secrets and include them in the deployment.
 For security reasons, your `.env` file is replaced with an empty file at runtime. Ploomber only stores your encrypted secrets.
 
+To learn how to read your secrets from within your application, see [Reading secrets.](../user-guide/secrets.md#reading-secrets)
+
 ```{note}
-Ensure that you specify all the required secrets in the `.env` file whenever deploying or re-deploying an application. Secrets will not be carried over from the previous deployment.
+By default, secrets are read from an `.env` file, but they may also be read from your [environment variables.](../user-guide/github.md#secrets)
```

Comment:
given my commment in the github.md file (see below) I think it makes sense to put a little guide here to show how you can read secrets from environment variables (in this case, we'll show the `export key=value` thing, what do you think?

it might be a little redundant but I think these are two separate cases:

1. when a user (for whatever reason wants to store secrets in env variables) -> user-guide/cli.md
2. when a user wants to deploy from github -> user-guide/github.md

user-guide/cli.md should link to user-guide/github.md and tell users that if they're using github actions for deployment they should look at that other tutorial

Reply:
I see what you're saying - that makes sense. I think the redundancy is fine because they're in different places and this is a more clear way of describing the two different use cases.

Reply:
Under `cli.md`, added section for [secrets without an .env file](https://github.com/ploomber/doc/pull/212/files#diff-bdba8f4b461f494edc3b95709bb0d5d869d0d6d1593c34c1ac790c9f24787e64R216-R253). This includes the `export key=value` workflow. Also added a note that if users are deploying via GitHub action, they should follow the link to `github.md`

`github.md` now better explains the GitHub secrets workflow.